### ISSUE TITLE
Why “No Pain, No Gain” Should Set Alarm Bells Ringing

### ISSUE THREAD

**Issue by JattMannu** [JattMannu/react-blog-github#23]
## The It’ll-Get-Worse-Before-It-Gets-Better Fallacy

A few years ago, I was on vacation in Corsica and fell sick. The symptoms were new to me, and the pain was growing by the day. Eventually I decided to seek help at a local clinic. A young doctor began to inspect me, prodding my stomach, gripping my shoulders and knees, and then poking each vertebra. I began to suspect that he had no idea what my problem was, but I wasn’t really sure so I simply endured the strange examination. To signal its end, he pulled out his notebook and said: “Antibiotics. Take one tablet three times a day. It’ll get worse before it gets better.” Glad that I now had a treatment, I dragged myself back to my hotel room with the prescription in hand.

The pain grew worse and worse—just as the doctor had predicted. The doctor must have known what was wrong with me after all. But, when the pain hadn’t subsided after three days, I called him. “Increase the dose to five times a day. It’s going to hurt for a while more,” he said. After two more days of agony, I finally called the international air ambulance. The Swiss doctor diagnosed appendicitis and operated on me immediately. “Why did you wait so long?” he asked me after the surgery.

I replied: “It all happened exactly as the doctor said, so I trusted him.”

“Ah, you fell victim to the it’ll-get-worse-before-it-gets-better fallacy. That Corsican doctor had no idea. Probably just the same type of stand-in you find in all the tourist places in high season.”

Let’s take another example: A CEO is at his wit’s end: Sales are in the toilet, the salespeople are unmotivated, and the marketing campaign sank without a trace. In his desperation, he hires a consultant. For $5,000 a day, this man analyzes the company and comes back with his findings: “Your sales department has no vision, and your brand isn’t positioned clearly. It’s a tricky situation. I can fix it for you—but not overnight. The measures will require sensitivity, and, most likely, sales will fall further before things improve.” The CEO hires the consultant. A year later, sales fall, and the same thing happens the next year. Again and again, the consultant stresses that the company’s progress corresponds closely to his prediction. As sales continue their slump in the third year, the CEO fires the consultant.

A mere smoke screen, the it’ll-get-worse-before-it-gets-better fallacy is a variant of the so-called confirmation bias. If the problem continues to worsen, the prediction is confirmed. If the situation improves unexpectedly, the customer is happy, and the expert can attribute it to his prowess. Either way he wins.

Suppose you are president of a country and have no idea how to run it. What do you do? You predict “difficult years” ahead, ask your citizens to “tighten their belts,” and then promise to improve the situation only after this “delicate stage” of “cleansing,” “purification,” and “restructuring.” Naturally you leave the duration and severity of the period open.

The best evidence of this strategy’s success is the religious zealot who believes that before we can experience heaven on earth, the world must be destroyed. Disasters, floods, fires, death—they are all part of the larger plan and must take place. These believers will view any deterioration of the situation as confirmation of the prophecy and any improvement as a gift from God.

In conclusion: If someone says, “It’ll get worse before it gets better,” you should hear alarm bells ringing. But beware: Situations do exist where things first dip, then improve. For example, a career change requires time and often incorporates loss of pay. The reorganization of a business also takes time. But in all these cases, we can see relatively quickly if the measures are working. The milestones are clear and verifiable. Look to these rather than to the heavens.
